docs/quick-start.md

```diff
@@ -358,7 +355,7 @@ kubectl exec -n vm $VMAGENT_POD_NAME -c vmagent  -- wget -qO -  http://127.0.0.1
 ```
 You’ll see demo-app in the list, along with a few other targets.
 The extra ones, like vmsingle-demo and vmagent-demo, are added automatically.
-That’s because the VictoriaMetrics operator creates scrape configs for its own components by default.
+That’s because the VictoriaMetrics operator creates scrape configs for its own resources by default.
```

Comment:
>for its own resources by default.

"own resources" feels a bit ambiguous, would be great to list the CRDs that has this features.

Reply:
fixed